packages/math-input/src/components/input/math-input.tsx

```diff
@@ -183,6 +188,8 @@ class MathInput extends React.Component<Props, State> {
             // multi-touch.
             if (this.state.focused && this.didTouchOutside && !this.didScroll) {
                 this.blur();
+                this.mathField.blur();
+                this.props.onBlur && this.props.onBlur();
```

Comment:
Just a minified version

```suggestion
               this.props.onBlur?.();
```